target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -540,6 +540,9 @@ wd,mynet-wifi-rangeextender)
 xiaomi,aiot-ac2350)
 	ucidef_set_led_switch "wan" "WAN" "blue:wan" "switch0" "0x02"
 	;;
+ditel,dt-acu100)
+	ucidef_set_led_netdev "wan" "WAN" "red:wan" "eth0"
+	;;
```

Comment:
`eth1` is set as WAN in **02_network**, but `eth0` is used for "WAN" LED here...

Reply:
Okay, i will change eth0 to eth1